backend/api/v1/auth.py

```
@@ -3,20 +3,34 @@
 from pydantic import BaseModel, EmailStr
 from typing import Optional, Dict, Any
 import uuid
-from backend.db.supabase_client import supabase_client
+from backend.db.supabase_client import supabase_client, supabase_anon
+from backend.core.auth import get_current_user, get_current_user_profile
 from backend.core.config import settings
```

Ruff (0.11.9)

`backend.core.config.settings` imported but unused

Remove unused import: `backend.core.config.settings`

(F401)